src/main/java/de/tum/in/www1/artemis/service/FeedbackService.java

```diff
@@ -33,28 +31,6 @@ public FeedbackService(ResultRepository resultService, Optional<ContinuousIntegr
         this.feedbackRepository = feedbackRepository;
     }
 
-    /**
-     * Checking if the result already has feedbacks if not try retrieving them from bamboo and create them. Having feedbacks create a JSONObject which will be converted into the
-     * bamboo format to fit the already used frontend system
-     *
-     * @param result for which the feedback is supposed to be retrieved
-     * @return a set of feedback objects including test case names and error messages
-     */
-    @Transactional
-    @Deprecated
-    public List<Feedback> getFeedbackForBuildResult(Result result) {
```

Comment:
please activate again and note in the Deprecation that this will be removed in September 2019

Reply:
I'll change it.